snap/snapcraft.yaml

```diff
@@ -1,8 +1,17 @@
 name: sublime-text
 adopt-info: sublime-text
 summary: A sophisticated text editor for code, markup and prose.
+title: Sublime Text
 description: |
-  A sophisticated text editor for code, markup and prose.
+  Sublime Text is a powerful multi-platform text editor with a clean, modern user interface.
+  Bringing wide support for programming languages, extensions, syntax-based features and many other useful features alongside low resource usage, Sublime Text proves itself to be a very competent solution for a vast set of environments and applications.
+  
+  Note - Sublime Text may be downloaded and evaluated for free, however a license must be purchased for continued use.
+contact: https://www.sublimetext.com/support
+issues: https://github.com/snapcrafters/sublime-text/issues
+source-code: https://github.com/snapcrafters/sublime-text
+website: https://snapcraft.io/publisher/snapcrafters
```

Comment:
This should point to the sublime text website. Users click this when they want more information about the application itself, not about us (snapcrafters)